README.md

```diff
@@ -58,7 +58,14 @@ Example of adding your validator keys to a zip file (node-0.zip):
         Running the script with the 'install' parameter will prompt for the following:
             - number of nodes to be ran on the machine
             - validator display name for each node (this will only be asked one time)
+    
+    [INSTALL MULTIKEY SQUAD]    
```

Comment:
rename to "multikey group" here and everywhere? This will align the naming with the docs and avoid confussions with the observing squad.